packages/data/lib/di/data_injector.dart

```diff
@@ -1,4 +1,48 @@
+import 'package:data/interceptor/interceptor.dart';
+import 'package:domain/repository/movie_repository.dart';
 import 'package:get_it/get_it.dart';
+import 'package:dio/dio.dart';
+import '../repository/movie_repository.dart';
+import '../service/api_base_service.dart';
+import '../service/service_payload.dart';
+import '../utils/const.dart';
 
 void initDataInjector() {
+  _initApiModule();
+  _initRepositoryModule();
+}
+
+void _initApiModule() {
+  GetIt.I.registerSingleton<Dio>(
+    _buildDio(),
+    instanceName: "Dio",
```

Comment:
now we have only one instance of Dio in container. no need in specifying instance name